test/test_prims.py

```diff
@@ -752,6 +759,29 @@ def fn1(x):
             )
             self.assertFalse(includes_aten_to_copy)
 
+    @ops([op for op in op_db if op.name in {"permute"}], allowed_dtypes=(torch.float,))
+    def test_decomposition_method_vararg(self, device, dtype, op):
+        # some ops have vararg variants for the methods. this tests it.
+        # we don't have tests for varargs in OpInfo, so we need to
+        # improvise this a bit.
+        # The rule for general functions (the special cases being e.g. tensor
+        # creation functions taking shapes) is that things can be vararg
+        # if the method has only one argument of sequence type.
+        # we might need to adjust things for the factory functions or
+        # have them do their own test
+        from torch.fx.experimental.proxy_tensor import make_fx
+        from torch._prims.context import TorchRefsMode
+
+        sample_inputs = op.sample_inputs(device, dtype, requires_grad=False)
+        for sample_input in sample_inputs:
+            a = sample_input.input
```

Comment:
What about acquiring the first sample input that has a non-empty sequence as `input`?

Reply:
Args, probably, but you mean instead of running all of them?

Reply:
Yep -- maybe just one run is enough?